tests/dummy_server/static/wait_and_check.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Check Test</title>
+</head>
+
+<body>
+    <div class="main-div">
+        <button class="btn" onclick="showButton()">click me</button>
+
+        <p class="text"></p>
```

Comment:
I think this element is not needed here (the original one was simply to show a change and easily check the element was clicked)